doc/source/tune-contrib.rst

```diff
@@ -0,0 +1,111 @@
+Contributing to Tune
+====================
+
+We welcome (and encourage!) all forms of contributions to Tune, including and not limited to:
+
+- Code reviewing of patches and PRs.
+- Pushing patches
+- Documentation and examples
+- Community participation in forums and issues.
+- Code readability and code comments to improve readability
+- Test cases to make the codebase more robust
+- Tutorials, blog posts, talks that promote the project.
+
+
+Setting up a development environment
+------------------------------------
+
+If you have Ray installed via pip (``pip install -U ray``), you can develop Tune locally without needing to compile Ray.
+
+
+First, you will need your own fork to work on the code. Press the Fork button on the `ray project page <https://github.com/ray-project/ray/>`__.
+Then, clone the project to your machine and connect your repository to the upstream (main project) ray repository.
+
+.. code-block:: shell
+
+    git clone https://github.com/your-user-name/ray.git [path to ray directory]
+    cd [path to ray directory]
+    git remote add upstream https://github.com/ray-project/ray.git
+
+
+Then, run RAY_DIRECTORY/python/ray/setup-dev.py `(also here on Github) <https://github.com/ray-project/ray/blob/master/python/ray/setup-dev.py>`__ script.
```

Comment:
Would it be easier to instruct users to install ray with `pip install -e [path to ray directory]`? That seems more standard way of setting up the dev package. Or is there some additional benefit from running `setup-dev.py`?

Reply:
Oh, `setup-dev.py` allows users to avoid building (on my machine and other machines, it's been difficult trying to build ray).

Reply:
Oh I see, that makes sense. I've never really had any problem building ray.